Lib/pdb.py

```diff
@@ -232,7 +232,7 @@ def __init__(self, completekey='tab', stdin=None, stdout=None, skip=None,
         try:
             import readline
             # remove some common file name delimiters
-            readline.set_completer_delims(' \t\n`@#$%^&*()=+[{]}\\|;:\'",<>?')
+            readline.set_completer_delims(' \t\n`@#%^&*()=+[{]}\\|;:\'",<>?')
```

Comment:
There are no tests in this PR. 

What could this change break?

Reply:
There's no test for completion, presumably because it's not easy to test. It would change the behavior when the user tries to do a complete (hit `<tab>` in most cases) and there's a `$` in the line. In that case, `$` would be not considered as a delimiter (the text to be completed would include `$`). I can't think of any real cases where this matters. Actually, I can't think of a case where the user uses `$` in their pdb commands.